bower.json

```diff
@@ -32,13 +31,7 @@
     "next-myft-client": "^6.4.2",
     "o-ads": "^4.0.0",
     "o-buttons": "^4.0.1",
-    "n-message-prompts": "^2.1.8",
```

Comment:
I didn't! I can only assume this happened when I asked Bower to persist my resolution decision.